src/functionalTest/java/uk/gov/hmcts/reform/userprofileapi/UserProfileFunctionalTest.java

```diff
@@ -443,8 +444,28 @@ public void invalidBearerTokenRequestsShouldReturn401() {
     }
 
     @Test
+    @ToggleEnable(mapKey = DELETE_USER_BY_ID_OR_EMAIL_PATTERN, withFeature = false)
+    public void deleteActiveUserByEmailPatternShouldReturnFailureWhenToggledOff() {
+        log.info("deleteActiveUserByEmailPatternShouldReturnFailureWhenToggledOff :: STARTED");
+
+        Response response = testRequestHandler
+                .sendDeleteWithoutBody(requestUri + "/users?emailPattern=@prdfunctestuser.com");
+
+        assertThat(HttpStatus.FORBIDDEN.value()).isEqualTo(response.statusCode());
+        assertThat(response.getBody().asString()).contains(CustomSerenityRunner.getFeatureFlagName().concat(" ")
+                .concat(FeatureConditionEvaluation.FORBIDDEN_EXCEPTION_LD));
+
+        log.info("deleteActiveUserByEmailPatternShouldReturnFailureWhenToggledOff :: ENDED");
+    }
+
+    @AfterClass
```

Comment:
Any methods that have been updated to `static` is because `@AfterClass` can only be applied to static method